zz_generated.manifests/test-reporting.yaml

```diff
@@ -65,6 +65,19 @@ spec:
     tests:
     - testName: '[sig-arch][OCPFeatureGate:Example] should only run FeatureGated test
         when enabled'
+  - featureGate: GatewayAPI
```

Comment:
Nice! Does it mean that those tests will be run only when the given feature gate is enabled?

Reply:
Yes, I think so. But need to check the jobs to confirm.